routing/src/frrmi.rs

```diff
@@ -0,0 +1,384 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Open Network Fabric Authors
+
+use mio::net::UnixStream;
+use std::io::{self, ErrorKind, Read, Write};
+use std::os::fd::AsRawFd;
+use std::str::from_utf8;
+use std::time::Instant;
+use std::{collections::VecDeque, time::Duration};
+use thiserror::Error;
+
+#[allow(unused)]
+use tracing::{debug, error, info, warn};
+
+#[derive(Error, Debug)]
+pub(crate) enum FrrErr {
```

Comment:
Since we are not using tokio, is it possible for this thread to get blocked on Tx in a way that causes issues (e.g., receiving new routes, etc.)?  Generally, if this thread blocks I presume something is wrong with FRR but the agent could be hosed but FRR ok.  What do we do in that case?  Again, does not need to be fixed for this PR but if this is an issue, we should create an issue and put it in the R1 release.  We can triage it out to R2 based on where we get timeline wise.